### PR TITLE
Update default calibration

### PR DESCRIPTION
**Pull request by jdebacker**
This PR updates the default calibration for the OG-USA model.  Changes include:

1. A fix to `macro_params.py`, which corrects double counting of social security payments when determining the $\alpha_G$ parameter.  As a result, $\alpha_G$ is significantly larger.
2. Changes default start year to 2025
3. Updates AIME breakpoints for 2024 values
4. Rounds numbers to 2-4 digits, depending on the parameter.

### CODE REVIEW

File: examples/run_og_usa.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ def main():
         "mtry_params": d["mtry_params"],
         "mean_income_data": d["mean_income_data"],
         "frac_tax_payroll": d["frac_tax_payroll"],
+        "r_gov_shift": [-0.005],
+        "r_gov_scale": [0.0],
```
Comment (rickecon) [inline examples/run_og_usa.py:66]: @jdebacker. Should these values for `r_gov_shift`, `r_gov_scale`, `zeta_K`, and `zeta_D` just be in the `ogusa_default_parameters.json` file so we don't have to update them in the baseline of the run script?

**Comment by rickecon** [comment]
@jdebacker. I just have the one question about updating values in the `ogusa_default_parameters.json` instead of updating them in the sample run script. The current status of this PR is good to go, but I think I would prefer moving those changes to the `ogusa_default_parameters.json` file. Let me know what you think, and I'll merge this.

**Comment by jdebacker** [comment]
@rickecon Thanks for the review.  I just made the suggested changes.

File: ogusa/ogusa_default_parameters.json [hunk 2]
```diff
@@ -33,29 +33,29 @@
...
-    "r_gov_shift": [0.011677698019801785],
+    "initial_debt_ratio": 0.99,
+    "r_gov_scale": [0.0],
+    "r_gov_shift": [-0.005, -0.007, -0.009, -0.01],
     "cit_rate": [[0.21]],
     "c_corp_share_of_assets": 0.55,
-    "adjustment_factor_for_cit_receipts": [0.30909090909090914],
-    "tau_c": [[0.0]],
+    "adjustment_factor_for_cit_receipts": [0.3090],
+    "tau_c": [[0.01]],
```
Comment (rickecon) [inline ogusa/ogusa_default_parameters.json:58]: @jdebacker. Why is `tau_c=0.01` in this default parameterization? Is this capturing average state sales tax?

Comment (rickecon) [reply ogusa/ogusa_default_parameters.json:58]: @jdebacker. Is this to capture national excise tax revenue?

Comment (jdebacker) [reply ogusa/ogusa_default_parameters.json:58]: I was basing it on CBO's value of "other" federal revenues, which are mostly excise taxes and tariffs and so I thought we'll modeled by a consumption tax.

**Review by rickecon — APPROVED** [review]

**Comment by rickecon** [comment]
@jdebacker. This looks great. Merging.

**Pull request merged**